src/Store/index.js

```diff
@@ -1,20 +1,24 @@
-import { createStore, combineReducers, applyMiddleware } from 'redux'
-import { composeWithDevTools } from 'redux-devtools-extension'
-import thunk from 'redux-thunk'
+import { configureStore } from '@reduxjs/toolkit'
+
+// import { createStore, combineReducers, applyMiddleware } from 'redux'
+// import { composeWithDevTools } from 'redux-devtools-extension'
+// import thunk from 'redux-thunk'
 
 import journalReducer from '../Reducers/journalReducers'
 import activeJournalReducer from '../Reducers/activeJournalReducers';
 import loginReducer from '../Reducers/loginReducers'
 
-const reducers = combineReducers({
-    journals: journalReducer,
-    activeJournal: activeJournalReducer,
-    login: loginReducer
+const store = configureStore({
+    reducer: {
+        journalReducer,
+        activeJournalReducer,
+        loginReducer
```

Comment:
Need entries reducer here, I accidentally deleted it during a merge conflict when we switched from combineReducer to configureStore